### PR TITLE
- Add example for secure token storage

### PR DESCRIPTION
**Pull request by H4ckerxx44**
## Summary
Add [secure_token_storage.py](https://discord.com/channels/881118111967883295/881315976052879451/881324925347061770)

## Checklist

- [ ] If code changes were made then they have been tested.
    - [ ] I have updated the documentation to reflect the changes.
- [ ] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [ ] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

File: examples/secure_token_storage.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+import os
+
+# There are other ways to load environment variables but this is one
+from dotenv import load_dotenv # https://pypi.org/project/python-dotenv/
+
+load_dotenv()
+token = os.getenv('BOT_TOKEN')
+bot.run(token)
+
+# load_dotenv reads from a file called .env in the same directory as the python files which should containing TOKEN="1234567890"
```
Comment (TAG-Epic) [inline examples/secure_token_storage.py:13]: This could be cleaned up and moved above the import.

**Review by TAG-Epic — CHANGES_REQUESTED** [review]

**Comment by H4ckerxx44** [comment]
I moved the comment just above load_dontenv() so it is clear what this does. Hope this is fine.

File: examples/secure_token_storage.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Your bot token should not be made public with your code. Otherwise anyone can sign in as your bot and mess with its servers.
+# You should load the token as an environment variable, then access it like this:
+
+import os
+
+# There are other ways to load environment variables but this is one
+from dotenv import load_dotenv # https://pypi.org/project/python-dotenv/
+
+# load_dotenv reads from a file called .env in the same directory as the python files which should containing TOKEN="1234567890"
```
Comment (TAG-Epic) [inline examples/secure_token_storage.py:9]: Maybe change
Which should containing

To

Which should look roughly like

**Review by TAG-Epic — CHANGES_REQUESTED** [review]

**Comment by H4ckerxx44** [comment]
based on: https://ludwig.guru/s/look+roughly+like, I am not native english but it seems to be used that way?

**Review by TAG-Epic — APPROVED** [review]
LGTM

**Pull request merged**